.github/workflows/ci.yml

```
@@ -94,6 +94,19 @@ jobs:
           poetry-version: "1.8.5"
       - name: "Linting: markdownlint"
         run: "poetry run invoke markdownlint"
+  check-compatibiility-matrix:
```

Should `check-in-docker` rely on this passing or do we want it to be informational for now?

```suggestion
  check-compatibility-matrix:
```